commands/qemu-guest-tools/guesttools.go

```diff
@@ -0,0 +1,129 @@
+package qemuguesttools
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/taskcluster/go-got"
```

Comment:
nit: when the package name differs from the directory name, I like to be explicit, e.g. `import got "github.com/taskcluster/go-got"` - just to aid with visual understanding of code - but not a biggy